src/ralph/dns/views.py

```diff
@@ -62,6 +64,10 @@ def post(self, request, *args, **kwargs):
                 messages.error(
                     request, _('An error occurred while updating a record')
                 )
+            if to_create and self.dnsaas.create_dns_records(to_create):
```

Comment:
imo we should improve flow of this form - when some of the rows are not saved in dnsaas, there will only be error message at the top. Is it possible to show these rows again with error message near to them?